### DISCUSSION TITLE
PAT Token For Private Pull Request

### DISCUSSION THREAD

**Discussion by AnaheimDucks16** [anuraghazra/github-readme-stats#1090] [Q&A] [answered]
What do I need to check when creating a PAT token for it to count my private pull request count?

**Comment by florianbussmann** [comment, marked as answer]
I already answered this question here https://github.com/anuraghazra/github-readme-stats/issues/1055

This is what needs to be checked

![image](https://user-images.githubusercontent.com/3610244/117886757-fa1a9880-b2af-11eb-8844-2617d00fe90c.png)

**Reply by AnaheimDucks16** [reply]
When I tried that, it showed I had 700+ commits all time when I have less than 500 commits.

**Reply by rickstaa** [reply]
@AnaheimDucks16  The solution proposed by @florianbussmann is correct. I think your problem is caused by something else. You can [create a bug](https://github.com/anuraghazra/github-readme-stats/issues) report for it.
